src/nl/rubensten/texifyidea/settings/LabelingCommandInformation.kt

```diff
@@ -0,0 +1,23 @@
+package nl.rubensten.texifyidea.settings
+
+data class LabelingCommandInformation(var commandName: String, var position: Int, var labelPrevCmd: Boolean) {
+    companion object {
+        /**
+         * Needed to get bck the object from the string in the xml file
+         */
+        @JvmStatic fun fromString(string: CharSequence): LabelingCommandInformation {
+            val parts = string.split(";")
+            if (parts.size != 3) {
+                throw IllegalArgumentException("String must contain exactly three parts")
+            }
```

Comment:
```kotlin
require(parts.size == 3) { "String must contain exactly three parts, got ${parts.size}" }
```

Kotlin has these nice contract functions :) Also add a 'got' part for easier debugging.

Reply:
fixed in commit ed34f48cd6dd8da4a8ed3d61d6cf44599f4bc7dd